internal/dag/status_test.go

```diff
@@ -2369,4 +2369,75 @@ func TestDAGStatus(t *testing.T) {
 		},
 	})
 
+	// issue 3197: Fallback and passthrough HTTPProxy directive should emit a config error
+	invalidFallbackCertificateSvc := &contour_api_v1.HTTPProxy{
```

Comment:
Maybe `tlsPassthroughAndFallback` would be a more descriptive name for the variable

Reply:
Done